parse_pymon.py

```diff
@@ -0,0 +1,31 @@
+"""
```

Comment:
You forgot to remove this file, now there's two copies included

Reply:
seems that the series of schoolboy mistakes continues in this PR, cheers @bouweandela  :man_facepalming: